src/main/java/org/openstreetmap/atlas/geography/atlas/sub/SubAtlasCreator.java

```diff
@@ -401,6 +401,166 @@ public SubAtlasCreator(final Atlas atlas)
         return Optional.ofNullable(result);
     }
 
+    @SuppressWarnings("squid:S3776")
```

Comment:
Hmmmm? What is that? How about we keep the sonar errors, so at least we can see where they are later? :D

Reply:
ok, sure!